src/exchanges/kraken-client.js

```diff
@@ -155,6 +167,33 @@ class KrakenClient extends BasicClient {
     this._debounceSend("unsub-ticker", this._tickerSubs, false, { name: "ticker" });
   }
 
+  /**
```

Comment:
trying to remain as aligned to current standards that were already defined for other subscriptions